tests/test_quantum_gradients.py

```diff
@@ -290,7 +296,8 @@ def test_CVOperation_with_heisenberg_and_no_params(self):
         gradient analytic method can still be used, and returns the correct result."""
         self.logTestName()
 
-        for cls in qml.ops.cv.all_ops:
+        for name in qml.ops._cv__ops__:
```

Comment:
how come this was necessary?

Reply:
See https://github.com/XanaduAI/pennylane/pull/232/files#r295094959